### DISCUSSION TITLE
Preset for PLA+

### DISCUSSION THREAD

**Discussion by ironsniper1** [supermerill/SuperSlicer#1375] [Community contribution]
Would it be possible to have PLA+ added as a preset? or a way to add it ourselves?

**Comment by supermerill** [comment]
yes, you can edit the vendor files [here](https://github.com/slic3r/slic3r-profiles), and a pull request will be send when you validate the changes.

Be sure to test them before.

To add it to a vendor file, 
* open the desired ini file (for example basic.ini or PrusaResearch.ini)
* add your `[filament:PLA+]` section after an exiting one
* don't forget to add an `inherits = ` to set the good default parameters
* write the settings that need to be changed vs the default ones
* change the ini config_version in the ini and add a new line in the .idx one to register the new version.
